premake5.lua

```diff
@@ -293,6 +300,7 @@ plugincommon()
 files {
     "src/vst2/**.cpp",
     "src/vst2/**.h",
+    "VST_SDK_2.4/public.sdk/source/vst2.x/**.cpp",
     "vst3sdk/public.sdk/source/vst2.x/**.cpp",
```

Comment:
this does not actually include anything to solve VST 2 compilation, does it?

Reply:
@kzantow how does one incorporate vstsdk2.4 and vst3sdk into the mix, they don't seem to be a part of this repo. do they require a secondary install?

Reply:
`git submodule init` ?

Reply:
it's not perfect. unclear licensing etc. it seems that Steinb0rg has completely dropped support for VST 2, which I have no idea how the license plays out in that case. but i found some stuff to make it work, barring being taken down

Reply:
Well, I was able to find a VST2.4SDK from somewhere and am linking to it outside the repo, for now, hoping to get a Surge.vst to compile and be able to do a `tracktion/pluginval` run on it just to find out if it can actually work with macOSX.